types/quill/index.d.ts

```diff
@@ -173,6 +175,8 @@ export class Quill implements EventEmitter {
     formatText(index: number, length: number, source?: Sources): DeltaStatic;
     formatText(index: number, length: number, format: string, value: any, source?: Sources): DeltaStatic;
     formatText(index: number, length: number, formats: StringMap, source?: Sources): DeltaStatic;
+    formatText(range: RangeStatic, format: string, value: any, source?: Sources): DeltaStatic;
+    formatText(range: RangeStatic, formats: StringMap, source?: Sources): DeltaStatic;
```

Comment:
so here is the line of code which supports this: https://github.com/quilljs/quill/blob/5eac092fc43311e8cc40cc3a3507ced09aafbe1b/core/quill.js#L459

there's no documentation about it as far as i can see, and they don't have it in their own test cases. but it has been like that for a few years (at least), so i guess this is fine.